ocaml/lambda/debuginfo.ml

```diff
@@ -72,6 +78,14 @@ module Scoped_location = struct
     in
     cons Sc_method_definition str
 
+(*
+  let enter_lazy ~scopes = cons Sc_lazy (str scopes)
+*)
+  let enter_lazy ~scopes = cons Sc_lazy (dot scopes "(lazy)")
```

Comment:
This is adding a double-paren `((lazy))`. (cf what enter_anonymous_function does)

Reply:
Ah, the commented code will be the final version -- this is just for debugging to make sure the scope is there.

Reply:
Surely the scope should be there regardless?

Reply:
Sure, but I thought the idea is that these wouldn't be in the textual version of the scope?